CHANGELOG.rst

```diff
@@ -7,6 +7,13 @@ Release Notes
 6.6.0
 =====
 
+.. _Release Notes_6.6.0_Known Issues:
+
+Known Issues
+------------
+
+- JMX garbage collection metrics `jvm.gc.parnew.time` and `jvm.gc.cms.count` got renamed to `jvm.gc.minor_collection_time` and `jvm.gc.major_collection_count` on some JMX integrations. Both metric names will be avilable after the root cause is fixed.
```

Comment:
typo: `available`

also, we can still put this section under the `prelude` IMO.

we could simplify and clarify what `Both metric names will be available after the root cause is fixed` means I think 🤔 

Maybe something like `Since this change on the name of these 2 metrics may affect your dashboards and monitors, these metrics will also be sent under their older names in a later version of the Agent`. What do you think?